src/main/java/hudson/plugins/git/extensions/impl/LocalBranch.java

```diff
@@ -27,7 +31,7 @@ public String getLocalBranch() {
     public static class DescriptorImpl extends GitSCMExtensionDescriptor {
         @Override
         public String getDisplayName() {
-            return "Check out to specific local branch";
+            return "Check out to specific local branch, null to use remote branch name.";
```

Comment:
I think that extra text is better placed in the online help. I'd rather not widen the menu for text that needs description in the help.